API ref being built for some protected docs

I think something has changed in the device_plugin_container - I now get this error in the build:
```
Generating docs for compound dronecore::Action...
Generating docs for compound dronecore::Device...
/home/ubuntu/DroneCore/install/include/dronecore/device_plugin_container.h:117: warning: Member init_plugins() (function) of class dronecore::DevicePluginContainer is not documented.
/home/ubuntu/DroneCore/install/include/dronecore/device_plugin_container.h:120: warning: Member deinit_plugins() (function) of class dronecore::DevicePluginContainer is not documented.
/home/ubuntu/DroneCore/install/include/dronecore/device_plugin_container.h:123: warning: Member _impl (variable) of class dronecore::DevicePluginContainer is not documented.
/home/ubuntu/DroneCore/install/include/dronecore/device_plugin_container.h:126: warning: Member _action_impl (variable) of class dronecore::DevicePluginContainer is not documented.
/home/ubuntu/DroneCore/install/include/dronecore/device_plugin_container.h:130: warning: Member _gimbal_impl (variable) of class 
```

I'm also getting some rubbish output in the Device class. That rubbish is something I didn't support in docs generator because we weren't exporting the protected attributes. I can fix the output if we need this, but I assume that we don't need these attributes

```
## Protected Attributes
[<Element 'type' at 0x7fd0f71daa50> _impl](#classdronecore_1_1_device_plugin_container_1a44e15ec0148e6a820bc1a62b48d3bc2a)

[<Element 'type' at 0x7fd0f717c390> _action_impl](#classdronecore_1_1_device_plugin_container_1a4b53b53f546f49284b2c39bac9566521)

[<Element 'type' at 0x7fd0f717c650> _gimbal_impl](#classdronecore_1_1_device_plugin_container_1a0d6012ed978cf8231b2a31fc56b388a7)
```

Thoughts?

--- a/autogenerate_plugin_container.cmake
+++ b/autogenerate_plugin_container.cmake
@@ -143,11 +143,11 @@ foreach(class_name ${plugin_class_names})
         "${PLUGIN_GET_STRING}    ${class_name} &${get_name}() { return ${member_name}; }\n\n")
 
     set(PLUGIN_MEMBER_STRING
-        "${PLUGIN_MEMBER_STRING}    /* @private */\n")
+        "${PLUGIN_MEMBER_STRING}    /** @private internal use only.*/\n")
     set(PLUGIN_MEMBER_STRING
         "${PLUGIN_MEMBER_STRING}    ${impl_class_name} *${impl_member_name};\n")
     set(PLUGIN_MEMBER_STRING
-        "${PLUGIN_MEMBER_STRING}    /** @private */\n")
+        "${PLUGIN_MEMBER_STRING}    /** @private internal use only.*/\n")
     set(PLUGIN_MEMBER_STRING
         "${PLUGIN_MEMBER_STRING}    ${class_name} ${member_name};\n")
 

--- a/include/device_plugin_container.h.in
+++ b/include/device_plugin_container.h.in
@@ -49,18 +49,18 @@ ${PLUGIN_GET_STRING}
     const DevicePluginContainer &operator=(const DevicePluginContainer &) = delete;
 
 protected:
-    /* @private. */
+    /** @private internal use only. */
     virtual void init_plugins();
 
-    /* @private. */
+    /** @private internal use only. */
     virtual void deinit_plugins();
 
-    /* @private. */
+    /** @private internal use only. */
     DeviceImpl *_impl;
 
 ${PLUGIN_MEMBER_STRING}
 
-    /* @private. */
+    /** @private internal use only. */
     std::vector<PluginImplBase *> _plugin_impl_list;
 };
 

